exam001/models/SampleButton.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  SampleButton.swift
+//  exam001
+//
+//  Created by Seif Ghotouk on 11/12/18.
+//  Copyright © 2018 Adnan Al-Akhrass. All rights reserved.
+//
+
+import UIKit
+
+class SampleButton: UIButton {
+    private var title: String?
```

Comment:
Why we use `private` instead of public?
What are the other access levels? please provide example for each one.